Core/TImeSeries/SmoothedTimeSeries.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Core.TimeSeries
+{
+    public sealed class SmoothedTimeSeries : TimeSeries
+    {
+        private List<double> SmoothedSeries { get; } = new List<double>();
+        public double SmoothingRatio { get; }
+        
+        public SmoothedTimeSeries(double smoothingRatio) => SmoothingRatio = smoothingRatio;
+
+        public SmoothedTimeSeries(List<Transaction> transactions, double smoothingRatio) : this(smoothingRatio) =>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this constructor to use a generic collection designed for inheritance.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4992784)